packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -210,7 +209,14 @@ class InfiniteScroller extends PolymerElement {
   }
 
   /**
-   * @private
+   * @return {number}
+   */
+  get bufferOffset() {
+    return this.root.querySelector('.buffer').offsetTop;
```

Comment:
Could we use `this._buffers[0]` here to avoid repetitive `querySelector()` calls?

Reply:
Done.